plugins/bazaar-backend/src/service/router.ts

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  errorHandler,
+  PluginDatabaseManager,
+  resolvePackagePath,
+} from '@backstage/backend-common';
+import express from 'express';
+import Router from 'express-promise-router';
+import { Logger } from 'winston';
+import { Config } from '@backstage/config';
+
+export interface RouterOptions {
+  logger: Logger;
+  database?: PluginDatabaseManager;
+  config: Config;
+}
+
+export async function createRouter(
+  options: RouterOptions,
+): Promise<express.Router> {
+  const { logger } = options;
+  const db = await options.database?.getClient();
+
+  logger.info('Initializing Bazaar backend');
+
+  const migrationsDir = resolvePackagePath(
+    '@backstage/plugin-bazaar-backend',
+    'migrations',
+  );
+
+  await db?.migrate.latest({
+    directory: migrationsDir,
+  });
+
+  const router = Router();
+  router.use(express.json());
+
+  router.get('/health', (_, response) => {
+    logger.info('PONG!');
+    response.send({ status: 'ok' });
```

Comment:
One little unexpected tidbit is that you want to replace all json sending in this file with `.json(`, not `.send(`. It takes care of some Content-Type concerns and such.